docs/operator-manual/ingress.md

```diff
@@ -32,15 +32,19 @@ metadata:
   name: argocd-server-cli
   namespace: argocd
 spec:
+  # NOTE: the port must be ignored if you have strip_matching_host_port enabled on envoy
   host: argocd.example.com:443
   prefix: /
-  service: argocd-server:443
+  service: argocd-server:80
+  regex_headers:
+    Content-Type: "^application/grpc.*$"
+  grpc: true
 ```
 
-Login with the `argocd` CLI using the extra `--grpc-web-root-path` flag for gRPC-web.
+Login with the `argocd` CLI:
 
 ```shell
-argocd login <host>:<port> --grpc-web-root-path /
+argocd login <host>
```

Comment:
It seems `--grpc-web-root-path /` wasn't really necessary in this case. Can you try just with `--grpc-web` flag with the previous mapping configuration?

Reply:
I can try, but it's worth mentioning that we'd have the same conceptual problem: I'd rather just have `argocd login <host>` Just Work©™ rather than having to internally document and tell our users to use some combination of flags to make it work.

Reply:
(well, to be more precise, we used these two mappings internally to avoid having any extra flags, so it's not blocking us or anything, but this is still something that I found more useful nonetheless :slightly_smiling_face:)

Reply:
I also like the idea of avoid the use of the extra-flag and provide a better user experience. Can you please just make sure that it works with  the `--grpc-web` flag as I explained in my other comment?